assets/mods/simplify/postload.js

```diff
@@ -33,27 +35,46 @@
 				}
 	
 				const original = asset.substr(baseDir.length + 7);
-				this.generatePatch(original, asset, 'File: ' + asset + '.patch');
+				this.generatePatch(original, asset).then((res) => {
```

Comment:
Missing `.catch`. Consider using async await instead

Reply:
I think that was meant to follow the function of the original code (generation failure == unprinted), but will implement as stated.

Reply:
...so async await here would break API.

Reply:
Not really since you can surround it with try and catch wich is the standard solution to replace a `.catch`

Reply:
no, I mean, it implied the function had to be async

Reply:
Making this function async is not a problem

Reply:
...that said generatePatches had no meaningful result anyway, so it probably isn't a big deal

Reply:
er, to clarify that, I mean it uses console.log and didn't return anything, so it being async just means there's an active bunch of async stuff running semi-backgrounded rather than immediate

Reply:
Changing this function should not have any side effects since it is not supposed to be used at runtime in an automated way. Furthermore, it has no actual return values besides the log messages you mentioned. I have no problem with changing the bahaviour of this function.